release-checks.sh

```diff
@@ -21,6 +21,9 @@
 set -eu
 RELEASE_VERSION=$1
 
+# Minimum Docker memory required to run Docker integration tests for release
+MIN_DOCKER_MEM=6.0
```

Comment:
Can we please indicate the unit of this value somehow (either in the comment or - IMHO even better - in the variable name)?

Reply:
https://github.com/elastic/rally/pull/914/commits/601eb0ebb08c31769662843d1cd5b3b9adde2c9f